### DISCUSSION TITLE
Remove purgecss ignore comments from build files

### DISCUSSION THREAD

**Discussion by matt-kinful** [tailwindlabs/tailwindcss#5563] [Help]
Is it possible to remove the `/* purgecss start ignore */` comments from build files without minifying?

**Comment by npearson72** [comment]
I've added this plugin config and it strips all comments:


```
      new CssMinimizerPlugin({
        minimizerOptions: {
          preset: [
            'default',
            {
              discardComments: { removeAll: true }
            }
          ]
        }
      }),
```

**Comment by matt-kinful** [comment]
Sorry, I've rephrased my original question above. I didn't mean **cssnano** specifically, but was referring to minifying in general. Is there a way to discard comments during build without minifying?
